web/src/cypress/support/graphql.ts

```diff
@@ -10,16 +12,12 @@ interface GraphQLResponse {
   errors: [any]
 }
 
-function graphql2(query: string, variables?: any): Cypress.Chainable<any> {
```

Comment:
i'm curious as to why it's removing function return types, is this a rule we can enable?